aws/rust-runtime/aws-inlineable/src/s3_request_id.rs

```diff
@@ -70,7 +70,6 @@ where
 }
 
 /// Applies the extended request ID to a generic error builder
-#[doc(hidden)]
 pub fn apply_extended_request_id(
```

Comment:
I think this one can be `pub(crate)`

Reply:
Addressed in [6433ca](https://github.com/smithy-lang/smithy-rs/pull/3226/commits/c6433ca7f2c351cf27a5c2570a32db0538e363e8).